tests/ApiExceptionTest.php

```diff
@@ -3,97 +3,100 @@
 use Illuminate\Http\Request;
 use Illuminate\Http\Response;
 use Illuminate\Support\Facades\Route;
+use Illuminate\Support\Facades\Validator;
```

Comment:
this unused import removed automatically by lint, but file `AppException.php` is not automatically linted. Should we check php-cs-fixer setup?